### PR TITLE
TEST: Multi upload

### PR DESCRIPTION
**Pull request by bigvaimarcella**
# Description
This Feature allow submitter submit list submission fron a json file

## Checklist

- [X ] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [ X] I have checked that unit tests pass locally with my changes
- [ X] I have run the npm script lint on the frontend and backend
- [ X] I have added tests that prove my fix is effective or that my feature works
- [X ] I have added necessary documentation (if appropriate)
- [ X] I have approval from the product owner for the contribution in this pull request

### CODE REVIEW

File: app/frontend/package.json [hunk 1]
```diff
@@ -35,16 +35,20 @@
     "bootstrap-scss": "^4.6.1",
     "core-js": "^3.23.3",
     "deep-freeze": "^0.0.1",
+    "export-from-json": "^1.7.2",
```
Comment (timisenco2015) [inline app/frontend/package.json:38]: What is the purpose of this package "export-from-json": "^1.7.2"

File: app/frontend/package.json [hunk 2]
```diff
@@ -35,16 +35,20 @@
...
     "formiojs": "^4.14.6",
     "keycloak-js": "^15.0.2",
     "material-design-icons-iconfont": "^6.7.0",
     "moment": "^2.29.3",
     "nprogress": "^0.2.0",
+    "papaparse": "^5.4.0",
     "qrcode.vue": ">= 1.7.0 < 3.0.0",
     "uuid": "^8.3.2",
     "vue": "^2.7.2",
+    "vue-blob-json-csv": "^0.1.7",
```
Comment (timisenco2015) [inline app/frontend/package.json:51]: What is the purpose of this package "vue-blob-json-csv": "^0.1.7"

File: app/config/default.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "db": {
-    "database": "chefs",
+    "database": "chefs_bulk",
```
[the remark this answers is not in the document]
Comment (bigvaimarcella) [reply app/config/default.json:3]: Sorry, this file always empty on my side .

File: app/frontend/package.json (hunk 1 above)
Comment (bigvaimarcella) [reply app/frontend/package.json:38]: We dont use this package any more we gonna  remove it

File: app/frontend/package.json (hunk 2 above)
Comment (bigvaimarcella) [reply app/frontend/package.json:51]: This package is used to convert json to CSV file , we use it in the app to generate report to the submitter in a csv file .

**Pull request merged**